src/components/Homepage/LendByCategory/LoanCategory.vue

```diff
@@ -72,22 +71,18 @@
 import _get from 'lodash/get';
 import _throttle from 'lodash/throttle';
 import LoanCardController from '@/components/LoanCards/LoanCardController';
-import categoryRowArrowsVisibleMixin from '@/plugins/category-row-arrows-visible-mixin';
 import KvLoadingSpinner from '@/components/Kv/KvLoadingSpinner';
 
-// TODO change card component used in this component and these values to fit 3 cards in row.
-const cardWidth = 320;
-const cardRightMargin = 10;
+// These values have to be the same as the values in src/components/LoanCards/LendHomepageLoanCard.vue
+const cardWidth = 305;
```

Comment:
I don't think it's worth going down a rabbit hole on this right now, but something to chew on. From what I see here, javascript is using px values, and CSS is using rem values. If the user has their browser font-size (not browser zoom) set to a non-default, the box will take up a different number of px due to remminess, but the JS will still be running on the original px. One thought is to get the width of the card on the page using JS instead of hardcoding it. 

Again, not something worth doing now, I'd like to see this kindof multi-slide functionality added as an improvement to KvCarousel in the future instead.

Reply:
This is a good point @ryan-ludwig